packages/codemods/src/codeMods/__tests__/personaToAvatar/propsMod.test.ts

```diff
@@ -0,0 +1,71 @@
+import { Project, SyntaxKind, JsxAttributeStructure } from 'ts-morph';
+import { renamePrimaryTextProp, renameRenderCoin } from '../../mods/PersonaToAvatar/PersonaToAvatar.mod';
+import { findJsxTag } from '../../utilities';
+const personaPath = '/**/__tests__/mock/**/persona/**/*.tsx';
+// @TODO ensure that props are not renamed for non fabric personas if they exist
+
+const personaPropsFile = 'mPersonaProps.tsx';
+const personaSpreadPropsFile = 'mPersonaSpreadProps.tsx';
+
+describe('Persona props mod tests', () => {
+  let project: Project;
+
+  beforeEach(() => {
+    project = new Project();
```

Comment:
Do you actually need a new project each time? (is it mutating anything?)

Reply:
Unfortunately, yes. It mutates the virtual files and there doesn't appear to be a way to revert them.